skyportal/facility_apis/kait.py

```diff
@@ -0,0 +1,125 @@
+import requests
+from requests.auth import HTTPBasicAuth
+
+from . import FollowUpAPI
+from baselayer.app.env import load_env
+
+from ..utils import http
+
+env, cfg = load_env()
+
+
+if cfg['app.kait.port'] is None:
+    KAIT_URL = f"{cfg['app.kait.protocol']}://{cfg['app.kait.host']}"
+else:
+    KAIT_URL = (
+        f"{cfg['app.kait.protocol']}://{cfg['app.kait.host']}:{cfg['app.kait.port']}"
+    )
+
+
+class KAITRequest:
+
+    """A dictionary structure for KAIT ToO requests."""
+
+    def _build_payload(self, request):
+        """Payload json for KAIT queue requests.
+
+        Parameters
+        ----------
+
+        request: skyportal.models.FollowupRequest
+            The request to add to the queue and the SkyPortal database.
+
+        Returns
+        ----------
+        payload: json
+            payload for requests.
+        """
+
+        for filt in request.payload["observation_choices"]:
+            assert filt in ["U", "g", "r", "i", "z", "B", "V", "R", "I"]
+
+        # The target of the observation
+        target = {
+            'name': request.obj.id,
+            'ra': request.obj.ra,
+            'dec': request.obj.dec,
+            'filters': "".join(request.payload["observation_choices"]),
+            'exposure': 'long',
```

Comment:
Will this always be a long exposure or are there other modes?

Reply:
I asked WeiKang and long is all he gave me.

Reply:
Ok went back and confirmed that short is an option, added!